src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/MySqlGQLSupportedTypesTests.cs

```diff
@@ -57,6 +57,8 @@ protected override bool IsSupportedType(string type)
             return type switch
             {
                 GUID_TYPE => false,
+                DATETIMEOFFSET_TYPE => false,
+                TIME_TYPE => false,
```

Comment:
i wonder if this should be an allow list so that arbitrary values don't accidentally pass a test because a value like "MYCUSTOMTYPE" would result in 'true' here, wouldn't it?

Reply:
If an arbitrary type is specified here and it passes, it means that db support that type, the main reason for adding these was to skip testing dbs for certain types.